src/domain/app/layout/utilityComponents/InfoTemplate.tsx

```diff
@@ -21,15 +23,19 @@ const InfoTemplate = ({
 }: Props) => {
   return (
     <div className={styles.infoPageLayout}>
-      <h1 className={styles.infoPageLayoutTitle}>{title}</h1>
+      <h1 className={styles.infoPageLayoutTitle}>
+        <HtmlToReact>{title}</HtmlToReact>
```

Comment:
The HtmlToReact should not be needed, since we can also use `\n` for line breaks. Those are already used elsewhere.

The HtmlToReact is too heavy for this. When dealing with translations, there is actually a way to do this with i18next component as well: https://react.i18next.com/latest/trans-component#usage-with-simple-html-elements-like-less-than-br-greater-than-and-others-v10.4.0.

Reply:
I suppose that the real HTML tags should not be used, but some custom HTML tags that describes the usage more, could be used. See an example from Kultus:
1. https://github.com/City-of-Helsinki/palvelutarjotin-ui/blob/master/public/locales/fi/newsletter.json#L19
2. https://github.com/City-of-Helsinki/palvelutarjotin-ui/blob/master/src/domain/newsletter/SubscribeNewsletterPage.tsx#L101-L119

Reply:
Switched to react-i18next Trans in 5af6d83c0a0798dbeddbfbec2f9f9cbf22214c70